src/app/secure-data-environment/sde-rest-handler.ts

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { ISdeRestHandler } from './sde-rest-handler.interface';
+import { Injectable } from '@angular/core';
+import { catchError, throwError } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SdeRestHandler implements ISdeRestHandler {
+  constructor(private httpClient: HttpClient) {}
+
+  get<T>(url: string): any {
+    return this.httpClient
+      .get<T>(url)
```

Comment:
We may need to think about some error codes that contain useful info for end users e.g. 422 will often say what problems prevented the user's action - but that won't be for "get"